modules/dxp/apps/scim/scim-rest-impl/rest-openapi.yaml

```diff
@@ -121,6 +136,7 @@ components:
                     description:
                         A multi-valued list of complex objects containing the requested resources.
                     oneOf:
+                        - $ref: "#/components/schemas/GroupSchema"
```

Comment:
Sadly not. In `BaseGroupResourceTestCase` and `BaseUserResourceTestCase` both `import com.liferay.scim.rest.client.dto.v1_0.Group;` and `import com.liferay.portal.kernel.model.Group;` will exist which causes a collision I cannot fix because even if I use one of them with its fully qualified form the next `gw buildRest` will overwrite it and switch back to the original which breaks the build. If you have any idea I'm open to it.

Reply:
I see now. Thanks for explaining.